fiona/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: fiona
+  version: {{environ['CBUILD_FIONA_VERSION']}}
+
+source:
+  git_tag: {{environ['CBUILD_FIONA_TAG']}}
+  git_url: https://github.com/Toblerity/Fiona
+
+build:
+  number: 1
+
+requirements:
+  build:
+    - python
+    - setuptools
```

Comment:
When adding `--single-version-externally-managed` it is usually unnecessary to list run dependencies in the build section.  That said, if the package imports itself to get a version or something, then you need to list them...